.goreleaser.yml

```diff
@@ -1,287 +1,40 @@
-    goarch:
-        - amd64
-    env:
-        - CGO_ENABLED=0
-    goos:
-        - linux
-    ldflags:
-      - -X "github.com/letsencrypt/boulder/core.BuildID={{ .ShortCommit }}"
-      - -X "github.com/letsencrypt/boulder/core.BuildHost={{ .Env.USER }}@{{ .Env.HOSTNAME }}"
-      - -X "github.com/letsencrypt/boulder/core.BuildTime={{ .Date }}"
-
-  - <<: *build
-    id: akamai-purger
-    main: ./cmd/akamai-purger
-    binary: akamai-purger
-
-  - <<: *build
-    id: bad-key-revoker
-    main: ./cmd/bad-key-revoker
-    binary: bad-key-revoker
-
-  - <<: *build
-    id: boulder-ca
-    main: ./cmd/boulder-ca
-    binary: boulder-ca
-    env:
-      - CGO_ENABLED=1
-
-  - <<: *build
-    id: boulder-observer
-    main: ./cmd/boulder-observer
-    binary: boulder-observer
-
-  - <<: *build
-    id: boulder-publisher
-    main: ./cmd/boulder-publisher
-    binary: boulder-publisher
-    env:
-      - CGO_ENABLED=1
-
-  - <<: *build
-    id: boulder-ra
-    main: ./cmd/boulder-ra
-    binary: boulder-ra
-    env:
-      - CGO_ENABLED=1
-
-  - <<: *build
-    id: boulder-sa
-    main: ./cmd/boulder-sa
-    binary: boulder-sa
-
-  - <<: *build
-    id: boulder-va
-    main: ./cmd/boulder-va
-    binary: boulder-va
-
-  - <<: *build
-    id: boulder-wfe
-    main: ./cmd/boulder-wfe
-    binary: boulder-wfe
-    env:
-      - CGO_ENABLED=1
-
-  - <<: *build
-    id: boulder-wfe2
-    main: ./cmd/boulder-wfe2
-    binary: boulder-wfe2
+  - id: "boulder-build"
+    main: ./cmd/boulder
     env:
+      - GO111MODULE=on
       - CGO_ENABLED=1
+      - GOFLAGS=-mod=vendor
+    binary: "boulder"
+    goos:
+      - linux
+    goarch:
+      - amd64
 
-  - <<: *build
-    id: caa-log-checker
-    main: ./cmd/caa-log-checker
-    binary: caa-log-checker
-
-  - <<: *build
-    id: ceremony
-    main: ./cmd/ceremony
-    binary: ceremony
-    env:
-      - CGO_ENABLED=1
-
-  - <<: *build
-    id: cert-checker
-    main: ./cmd/cert-checker
-    binary: cert-checker
-
-  - <<: *build
-    id: contact-auditor
-    main: ./cmd/contact-auditor
-    binary: contact-auditor
-
-  - <<: *build
-    id: expiration-mailer
-    main: ./cmd/expiration-mailer
-    binary: expiration-mailer
-
-  - <<: *build
-    id: id-exporter
-    main: ./cmd/id-exporter
-    binary: id-exporter
-
-  - <<: *build
-    id: log-validator
-    main: ./cmd/log-validator
-    binary: log-validator
-
-  - <<: *build
-    id: nonce-service
-    main: ./cmd/nonce-service
-    binary: nonce-service
-
-  - <<: *build
-    id: notify-mailer
-    main: ./cmd/notify-mailer
-    binary: notify-mailer
-
-  - <<: *build
-    id: ocsp-responder
-    main: ./cmd/ocsp-responder
-    binary: ocsp-responder
-    env:
-      - CGO_ENABLED=1
+# https://goreleaser.com/customization/checksum
+checksum:
+  name_template: "checksums.txt"
 
-  - <<: *build
-    id: ocsp-updater
-    main: ./cmd/ocsp-updater
-    binary: ocsp-updater
+# https://goreleaser.com/customization/changelog
+changelog:
+  skip: true
```

Comment:
If we're going to be creating full releases anyway, I wouldn't mind having a changelog. Are we skipping it here for a specific reason, or just because we haven't had one in the past?

Reply:
Recap of offline discussion:

GitHub releases are for internal use only. We plan remove any prior release before issuing a new one. The version and corresponding tag will always be `v0.0.0`